dts/bindings/vendor-prefixes.txt

```diff
@@ -72,6 +73,7 @@ arm	ARM Ltd.
 armadeus	ARMadeus Systems SARL
 arrow	Arrow Electronics
 artesyn	Artesyn Embedded Technologies Inc.
+arturo182	Arturo182
```

Comment:
This vendor should like be "solderparty" or similar, arturo182 is just his username. Another option would be to put them under "others".

https://www.tindie.com/stores/arturo182/

Reply:
Fixed